src/routes/safe/components/Transactions/TxsTable/columns.js

```diff
@@ -46,7 +46,7 @@ export const getTxAmount = (tx: Transaction) => {
 
   if (tx.isTokenTransfer && tx.decodedParams) {
     const tokenDecimals = tx.decimals.toNumber ? tx.decimals.toNumber() : tx.decimals
-    txAmount = `${new BigNumber(tx.decodedParams.value).div(10 ** tokenDecimals).toString()} ${tx.symbol}`
+    txAmount = `${new BigNumber(tx.decodedParams.value).div(10 ** tokenDecimals).toFixed()} ${tx.symbol}`
```

Comment:
why was `toString()` replaced with `toFixed()`?

Reply:
we should use formatAmount util for formatting numbers
```
safe-react/src/logic/tokens/utils/formatAmount.js
```

Reply:
how was the amount rendered before:
![image](https://user-images.githubusercontent.com/3315606/80211403-0cc22100-860c-11ea-8ef6-133d338e4bbc.png)
<br>
so we went with https://mikemcl.github.io/bignumber.js/#toFix:
![image](https://user-images.githubusercontent.com/3315606/80211470-32e7c100-860c-11ea-8b5d-69f384d18aff.png)

<br><br>
Now I tried with:

```TypeScript
    txAmount = `${BigNumber(tx.decodedParams.value}.div(10 ** tokenDecimals).toString()} ${tx.symbol}`
```
...but it didn't work either. It looks like the... https://github.com/gnosis/safe-react/blob/2a8ec9d4db39edd055c36e3e51dcd5e315203ae7/src/index.js#L14
...is not making any effect.

<br><br>
If you know the reasons why `EXPONENTIAL_AT` setup is not working, please help.

Reply:
if you'd use `formatAmount` it wouldn't be a problem :) 
also,  `BigNumber.set({ EXPONENTIAL_AT: [-7, 255] }) ` so the lowest to switch to exponential is -7, and on your screenshot the amount is exactly 1e-7

Reply:
> we should use formatAmount util for formatting numbers
> 
> ```
> safe-react/src/logic/tokens/utils/formatAmount.js
> ```

@mikheevm, ok. I completely missed that function. Incredible :-|

Reply:
this is the guide for formatting amounts:
https://github.com/gnosis/safe/blob/a87d5d8af99a4d6fd35424a7e0743379a44bf9a8/archive/specs/common/format_amounts.rst

Reply:
what happens if you change ` BigNumber.set({ EXPONENTIAL_AT: [-7, 255] }) ` to ` BigNumber.set({ EXPONENTIAL_AT: [-19 or -18, 255] }) `?

Reply:
wrapping `toString()` result with `formatAmount` gives:
![image](https://user-images.githubusercontent.com/3315606/80213142-1e58f800-860f-11ea-9b01-c3d4c6fe7006.png)

And that's the expected result, isn't it?

Reply:
yes, just one thing
@lukasschor is this the expected formatting for transaction list?

Reply:
Do we show the exact number in the transaction details? If yes, then I think this should be fine.

Before we had the exact amount also in the overview, but I don't think this makes sense.

![image](https://user-images.githubusercontent.com/17196842/80215259-74906000-863c-11ea-9e2d-07ec46a0a603.png)